js_modules/dagster-ui/packages/ui-core/src/runs/RunsFilterInput.tsx

```diff
@@ -395,166 +393,214 @@ export const useRunsFilterInput = ({tokens, onChange, enabledFilters}: RunsFilte
+        icon = <Icon name="schedule" />;
+      } else if (value.type === DagsterTag.User) {
+        return <UserDisplay email={value.value!} isFilter />;
+      } else if (value.type === DagsterTag.Automaterialize) {
+        icon = <Icon name="auto_materialize_policy" />;
+        labelValue = 'Auto-materialize policy';
+      }
+      return (
+        <Box flex={{direction: 'row', gap: 4, alignItems: 'center'}}>
+          {icon}
+          <TruncatedTextWithFullTextOnHover text={labelValue!} />
+        </Box>
+      );
+    },
+    getStringValue: (x) => {
+      if (x.type === DagsterTag.Automaterialize) {
+        return 'Auto-materialize policy';
+      }
+      return x.value!;
+    },
+    initialState: React.useMemo(() => {
+      return new Set(
+        tokens
+          .filter(
+            ({token, value}) =>
+              token === 'tag' && CREATED_BY_TAGS.includes(value.split('=')[0] as DagsterTag),
+          )
+          .map(({value}) => tagValueToFilterObject(value)),
+      );
+    }, [tokens]),
+    onStateChanged: (values) => {
+      onChange([
+        ...tokens.filter((token) => {
+          if (token.token !== 'tag') {
+            return true;
+          }
+          return !CREATED_BY_TAGS.includes(token.value.split('=')[0] as DagsterTag);
+        }),
+        ...Array.from(values).map((value) => ({
+          token: 'tag' as const,
+          value: `${value.type}=${value.value}`,
+        })),
+      ]);
+    },
+  });
+
+  const createdDateFilter = useTimeRangeFilter({
+    name: 'Created date',
+    icon: 'date',
+    initialState: React.useMemo(() => {
+      const before = tokens.find((token) => token.token === 'created_date_before');
+      const after = tokens.find((token) => token.token === 'created_date_after');
+      return [
+        after ? parseInt(after.value) * 1000 : null,
+        before ? parseInt(before.value) * 1000 : null,
+      ] as TimeRangeState;
+    }, [tokens]),
+    onStateChanged: (values) => {
+      onChange([
+        ...tokens.filter(
+          (token) => !['created_date_before', 'created_date_after'].includes(token.token ?? ''),
+        ),
+        ...([
+          values[0] != null ? {token: 'created_date_after', value: `${values[0] / 1000}`} : null,
+          values[1] != null ? {token: 'created_date_before', value: `${values[1] / 1000}`} : null,
+        ].filter((x) => x) as RunFilterToken[]),
+      ]);
+    },
+  });
+
+  const ID_EMPTY = 'Type or paste 36-character ID';
+  const ID_TOO_SHORT = 'Invalid Run ID';
+
+  const idFilter = useSuggestionFilter({
+    name: 'Run ID',
+    icon: 'id',
+    initialSuggestions: [{final: true, value: ID_EMPTY}],
+    state: React.useMemo(() => {
+      return tokens.filter(({token}) => token === 'id').map((token) => token.value);
+    }, [tokens]),
+    freeformSearchResult: (query) => {
+      return /^([a-f0-9-]{36})$/.test(query.trim()) ? {value: query.trim(), final: true} : null;
+    },
```

Comment:
We should add an "emptyPlaceholder" prop to the base filter

Reply:
to make it work I would make FilterDropdown.tsx call getPlaceholder() or something like that on the `selectedFilter` and pass it the current search query. The filter can return null if they don't want to show a placeholder or return a placeholder. That way we don't make the message clickable.

Reply:
This sounds good @salazarm - will make this change!